eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -35,7 +35,7 @@ steps:
         CommitMsg: ${{ parameters.CommitMessage }}
         PRTitle: ${{ parameters.PRTitle }}
         PRBody: ${{ parameters.PRBody }}
-        PRLabels: "EngSys"
+        PRLabels: "Central-EngSys"
```

Comment:
I think we want both EngSys and Central-EngSys

Reply:
Reading the rest api docs at https://developer.github.com/v3/issues/#update-an-issue

>Labels to associate with this issue. Pass one or more Labels to replace the set of Labels on this Issue. Send an empty array ([]) to clear all Labels from the Issue. NOTE: Only users with push access can set labels for issues. Labels are silently dropped otherwise.

Looks like it always replaces the set of labels so we would need to query the labels first if we wanted to just add one.

Reply:
Although in our context we either created or got a pull request https://developer.github.com/v3/pulls/#get-a-pull-request and so I think the labels will be part of that payload already.

Reply:
@sima-zhu @chidozieononiwu FYI we will need to make sure to ensure we aren't resetting the labels/reviewers/assignees when we already have an existing PR open.

Reply:
#1046 is tracking the resetting of existing labels issue